CHANGELOG.md

```diff
@@ -2,12 +2,12 @@
 
 ### Enhancements
 * Added a new flag to `CollectionChangeSet` to indicate when collections are cleared. ([#5340](https://github.com/realm/realm-core/pull/5340))
+* Added auth code and id token support for google c_api ([#5347](https://github.com/realm/realm-core/issues/5347))
```

Comment:
A rebase problem here?

Reply:
This changelog entry was in the wrong section and had invalid syntax.